docs/EN_US/ContainerizedHPCC/ContainerizedMods/PVCStorage.xml

```diff
@@ -105,8 +105,11 @@ helm install hpcc-localfile examples/local/hpcc-localfile --set common.hostpath=
         <para>The example directory contains a sample yaml file that can be
         used in this case:</para>
 
-        <programlisting>helm install mycluster hpcc/ --set global.image.version=latest 
+        <programlisting>helm install mycluster hpcc/ --version=8.2.2  
```

Comment:
the --version isn't necessary, but optional.
If it is not supplied it will install the latest (non-dev) release that is in the local repo (see other comment re. helm repo update)